docs/api.md

```diff
@@ -644,6 +644,33 @@ most.from([1, 2])
 	.observe(console.log.bind(console));
 ```
 
+### flatMapEnd
+
+####`stream.flatMapEnd(f) -> Stream`
+####`most.flatMapEnd(f, stream) -> Stream`
+
+Map the end event to a new stream, and begin emitting its values.  Note that `f` *must* return a stream.
```

Comment:
Maybe _extending_ the tail end of one stream with the start of a new one? :stuck_out_tongue_winking_eye: 
> "Replace the end signal with a new stream returned by f. Note that f must return a stream."

nicely put... i like it